src/effects/native/reverbeffect.cpp

```diff
@@ -77,7 +77,7 @@ EffectManifest ReverbEffect::getManifest() {
 
 ReverbEffect::ReverbEffect(EngineEffect* pEffect,
                              const EffectManifest& manifest)
-        : m_pSendParameter(pEffect->getParameterById("send")),
+        : m_pSendParameter(pEffect->getParameterById("send_amount")),
```

Comment:
Oh was the segfault just a null pointer? I was scratching my head about why calling init() twice could segfault -- the worst I can see that happens is we leak some memory in Delay if calling init twice.